devel/vulture/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                vulture
 set _n              [string index ${name} 0]
```

Comment:
you can remove this line and the `master_sites` and `distname` fields; these are set by the python PG